vital/config/config_block_formatter.cxx

```diff
@@ -94,24 +94,5 @@ print( std::ostream& str )
     str << std::endl;
   }
 }
-
-// ------------------------------------------------------------------
```

Comment:
It requires plugin_manager, which can't be built statically because it uses singleton. Also documented here: #862 under build process caveats

@as6520 knows more about the issue

Reply:
This method has been here for quite a while (years?) so it is probably not the root problem. I added a config print to pipeline_runner which probably caused the problem. I would rather remove the config print from the pipeline runner than delete this method.

Reply:
It was the method that was causing the issue. Since it `plugin_manager` a dependency for config. Originally I had removed it since I was not able to find any usage of the function in kwiver.